### ISSUE TITLE
Reinforcement learning! 

### ISSUE THREAD

**Issue by Bartleby2718** [Bartleby2718/DieOrDare#70]
- Useful Links
  - [QMaze (Very useful)](https://www.samyzaf.com/ML/rl/qmaze.html)
  - [TensorFlow docs](https://www.tensorflow.org/guide/tensors)
  - [Keras docs](https://keras.io)
  - Others: https://medium.com/@dhruvp/how-to-write-a-neural-network-to-play-pong-from-scratch-956b57d4f6e0 https://www.analyticsvidhya.com/blog/2017/01/introduction-to-reinforcement-learning-implementation/

- Memo
  - negative reward when the next duel starts 
  - AI로 할 것 : joker value, joker position, deck choice, action
  - reward -1 ~ 1
  - chance oc reward
  - new game once total reward fall below a negative threshold
